src/pm/hydra/ui/mpich/utils.c

```diff
@@ -903,7 +903,7 @@ static void bind_to_help_fn(void)
     printf("            user:0+2,1+4,3,2 -- user specified binding\n");
     printf("\n");
     printf("        Architecture aware options (part within the {} braces are optional):\n");
-    printf("            board{:<n>}      -- bind to 'n' motherboards\n");
+    printf("            machine{:<n>}      -- bind to 'n' motherboards\n");
```

Comment:
Help message needs improvement.  Also, is `{:<n>}` appropriate anymore?

Reply:
Will remove the :<n> for machine binding option, thank you.

Reply:
It's a question, @KavithaTipturMadhu.

Reply:
sorry, it is not relevant anymore since the newer version of hwloc no longer supports system objects and machine is the topmost hardware object that canbe discovered.

Reply:
Aren't multiple "machines" supported anymore?  I thought `hwloc` supported a model at some point where multiple non-cache-coherent entities could be tied together.

Reply:
Sorry, I think I misunderstood the release notes where they said "system" objects are no longer supported. The documentation says _Additionally it may assemble
the topologies of multiple machines into a single one so as to let applications consult the topology of an entire fabric or cluster at once_. I am not sure if the query used in hydra creates a parent node to machine objects in the topology tree. I will verify this and get back to you. Thank you.